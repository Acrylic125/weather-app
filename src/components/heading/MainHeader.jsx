import React from "react";
import cx from "classnames";

export default function MainHeader({ children, ...otherProps }) {
  return (
    <h1
      style={{
        fontWeight: "bold",
        color: "#ffffff",
      }}
      {...otherProps}
    >
      {children}
    </h1>
  );
}
// export default function MainHeader({ children, className, ...otherProps }) {
//   return (
//     <h1 className={cx("font-bold text-white", className)} {...otherProps}>
//       {children}
//     </h1>
//   );
// }
