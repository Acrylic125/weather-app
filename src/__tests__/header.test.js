import React from "react";
import testRenderer from "react-test-renderer";
import MainHeader from "../components/heading/MainHeader";

describe("Given the MainHeader component", () => {
  it("should render correctly", () => {
    const tree = testRenderer.create(<MainHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
