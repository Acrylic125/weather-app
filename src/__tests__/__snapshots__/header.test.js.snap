// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the MainHeader component should render correctly 1`] = `
<h1
  style={
    Object {
      "color": "#ffffff",
      "fontWeight": "bold",
    }
  }
/>
`;
